src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styleString) {
+  const styleObject = {};
+  const lines = styleString.split(';');
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':');
+
+    if (property && value) {
+      const trimmedProperty = property.trim();
+      const trimmedValue = value.trim();
+
+      if (trimmedProperty && trimmedValue) {
+        styleObject[trimmedProperty] = trimmedValue;
+      }
```

Comment:
The nested if statements (line 8 and line 12) are redundant since both conditions are checking if 'property' and 'value' are truthy after being trimmed. You can combine these conditions into a single if statement to simplify the code.